starter-code/src/viking.js

```diff
@@ -1,11 +1,48 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+        Soldier.prototype.attack = function attack() {return this.strength}
+        Soldier.prototype.receiveDamage = function(damage) {this.health -= damage}
+        }
+    }
+
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength) {
+        super(health,strength);
+        this.name = name;
+        Viking.prototype.receiveDamage = function receiveDamage(damage) {
+            this.health -= damage;
```

Comment:
Here you could have called the parents function by using, super.receiveDamage(damage)
For this case it's not a problem since there is only 1 line of code, but if there was more logic to the function you
would be writing it over again.